src/genesis/mod.rs

```diff
@@ -186,17 +186,20 @@ where
         },
     );
 
+    let chunk_size = std::env::var("CHUNK_SIZE")?.parse::<usize>()?;
```

Comment:
Why not CLI args instead? Would probably be good to have some reasonable defaults too?